README.md

```diff
@@ -63,6 +63,17 @@ client = CerberusClient('https://my.cerberus.url', role_arn='arn:aws:iam::000000
 ```
 ** Note: In this case, the client authenticates with Cerberus using the given role, then tries to assume that role in order to decrypt the Cerberus auth payload.
 
+IAM Role Authentication with IAM Role ARN(Lambda):
+```python
+client = CerberusClient('https://my.cerberus.url', role_arn='arn:aws:iam::0000000000:role/role-name', assume_role=False)
+```
+** Note: Unlike the default IAM Role Authentication constructor, this constructor will not attempt to figure out the role_arn, which may decrease overall latency. This the recommended constructor for Lambda.
+
+IAM Role Authentication with Lambda context
+```python
+client = CerberusClient('https://my.cerberus.url', lambda_context=context)
+```
+** Note: This constructor adds extra latency to your Lambda.
```

Comment:
I think this is only worth mentioning if there is a significant amount of latency added. Do we know how much it adds?